### PR TITLE
Made buttons reset upon submitting

### PR DESCRIPTION
**Pull request by ggambhirdev**
Yeah I missed a letter in the branch name by accident. Buttons should now reset to "no" or "none" after data is submitted

### CODE REVIEW

File: src/Components/teleop-page/Stage.tsx [hunk 1]
```diff
@@ -16,92 +16,56 @@ function Stage() {
...
-	const [none, setNone] = useState<'outlined' | 'contained'>('outlined');
-	const [parked, setParked] = useState<'outlined' | 'contained'>('outlined');
-	const [single, setSingle] = useState<'outlined' | 'contained'>('outlined');
-	const [double, setDouble] = useState<'outlined' | 'contained'>('outlined');
-	const [triple, setTriple] = useState<'outlined' | 'contained'>('outlined');
+	const getButtonStyle = (status) => {
+		if (stageValue === stageValues[status]) {
+			return 'contained';
+		}
+			return 'outlined';
```
Comment (PatrickUbelhor) [inline src/Components/teleop-page/Stage.tsx]: This appears indented one too many times

File: src/Components/auto-page/Leave.tsx [hunk 2]
```diff
@@ -11,37 +11,36 @@ function Leave() {
 	const dispatch = useAppDispatch();
 	const leaveValue = useAppSelector(state => state.auto.mobility);
 
-	const [no, setNo] = useState<'outlined' | 'contained'>('outlined');
-	const [yes, setYes] = useState<'outlined' | 'contained'>('outlined');
+	const getButtonStyle = (status) => {
+		if (leaveValue === leaveValues[status]) {
+				return 'contained';
```
Comment (PatrickUbelhor) [inline src/Components/auto-page/Leave.tsx]: This appears indented one too many times

**Review by PatrickUbelhor — CHANGES_REQUESTED** [review]

File: src/Components/auto-page/Leave.tsx (hunk 2 above)
Comment (ggambhirdev) [reply src/Components/auto-page/Leave.tsx]: Fixed with [49f37ff](https://github.com/Team2338/GearScout2024/pull/4/commits/49f37ffe19aabe3d561ecc34df59aa6e7abdfd30)

File: src/Components/teleop-page/Stage.tsx (hunk 1 above)
Comment (ggambhirdev) [reply src/Components/teleop-page/Stage.tsx]: Fixed with [49f37ff](https://github.com/Team2338/GearScout2024/pull/4/commits/49f37ffe19aabe3d561ecc34df59aa6e7abdfd30)

**Review by PatrickUbelhor — APPROVED** [review]

**Pull request merged**